Events/PostSyncEvent.lua

```diff
@@ -0,0 +1,58 @@
+--Post sync is used for variables that need predefined variables in all vehicles 
+
+PostSyncEvent = {}
+PostSyncEvent_mt = Class(PostSyncEvent, Event)
+
+InitEventClass(PostSyncEvent, "PostSyncEvent")
+
+function PostSyncEvent:emptyNew()
+	local self = Event:new(PostSyncEvent_mt)
+	self.className = "PostSyncEvent"
+	return self
+end
+
+function PostSyncEvent:new(vehicle)
+	self.vehicle = vehicle
+	return self
+end
+
+function PostSyncEvent:writeStream(streamId, connection)
+	streamWriteInt32(streamId, NetworkUtil.getObjectId(self.vehicle))
+	---assignedCombines
+	if self.vehicle.cp.assignedCombines then 
+		print("assignedCombines found")
+		for combine,data in pairs(self.vehicle.cp.assignedCombines) do
+			if data == true then 
+				print("write Combine: "..tostring(combine))
+				streamDebugWriteBool(streamId, true)
+				NetworkUtil.writeNodeObjectId(streamId, NetworkUtil.getObjectId(combine))
+			end
+		end
+	end
+	streamDebugWriteBool(streamId, false)
+end
+
+function PostSyncEvent:readStream(streamId, connection)
+	print("PostSyncEvent readStream")
```

Comment:
a debug would probably be better not to flood the logs (like self:debug()?)

Reply:
This one is leftover code from my early testing runs, a question would be how can we enable debug before the game is even started?